service/tutorial/order/OrderServices.xml

```diff
@@ -0,0 +1,165 @@
+            <iterate list="ordersList" entry="orderInfo">
+                <if condition="orderInfo.grandTotal &gt; 50">
+                    <then>
+                        <set field="orderCount" from="orderCount+[orderId:orderInfo.orderId,orderName:orderInfo.orderName,
+                        placedDate:orderInfo.placedDate,statusId:orderInfo.statusId,currencyUomId:orderInfo.currencyUomId
+                       ,productStoreId:orderInfo.productStoreId,grandTotal:orderInfo.grandTotal ]"/>
+                    </then>
+                </if>
+            </iterate>
+        </actions>
+    </service>
+
+
+    <service verb="get" noun="OrderItems" displayName="get the grand_total details">
+        <in-parameters>
+            <parameter name="orderId" default-value="100102"/>
+            <parameter name="orderPartSeqId" default-value="01"/>
+        </in-parameters>
+        <out-parameters>
+            <parameter name="orderItemList" type="list" required="true"/>
+        </out-parameters>
+        <actions>
+            <entity-find entity-name="mantle.order.OrderItem" list="orderItemList">
+                <econdition field-name="orderId"/>
+                <econdition field-name="orderPartSeqId"/>
+            </entity-find>
+        </actions>
+    </service>
+
+    <service verb="get" noun="OrderParts" displayName="get the part_total details">
+        <out-parameters>
+            <parameter name="entityCount" type="list" required="true"/>
+        </out-parameters>
+        <actions>
+            <set field="entityCount" from="[]"/>
+            <entity-find entity-name="mantle.order.OrderPart" list="ordersPartList">
+            </entity-find>
+            <iterate list="ordersPartList" entry="entityInfo">
+                <if condition="entityInfo.partTotal &lt; 20">
+                    <then>
+                        <set field="entityCount" from="entityCount+[orderId:entityInfo.orderId
+                        ,orderPartSeqId:entityInfo.orderPartSeqId,partName:entityInfo.partName,statusId:entityInfo.statusId
+                        ,vendorPartyId:entityInfo.vendorPartyId,customerPartyId:entityInfo.customerPartyId
+                        ,facilityId:entityInfo.facilityId,partTotal:entityInfo.partTotal]"/>
+                    </then>
+                </if>
+            </iterate>
+        </actions>
+    </service>
+
+    <service verb="get" noun="OrderPartData" displayName="get the order part details">
+        <in-parameters>
+            <parameter name="facilityId" default-value="ZIRET_WH"/>
+        </in-parameters>
+        <out-parameters>
+            <parameter name="orderPartLists" type="list" required="true"/>
+        </out-parameters>
+        <actions>
+            <entity-find entity-name="mantle.order.OrderPart" list="ordersPartList">
+                <econdition field-name="facilityId"/>
+            </entity-find>
+            <set field="orderPartLists" from="[orderId:ordersPartList.orderId
```

Comment:
Can you check if there is a way where we can select some fields in entity-find itself? You can explore the entity-find definition for this and check for it's possible attributes.

Reply:
Updated recommended changes.
Thank you.